site/source/docs/porting/connecting_cpp_and_javascript/embind.rst

```diff
@@ -218,6 +223,16 @@ Consider the example below:
             .field("age", &PersonRecord::age)
             ;
 
+		value_object<ArrayInStruct>("ArrayInStruct")
+			.field("field", &ArrayInStruct::field) // Need to register the array type
+			;
+
+		// Register std::array<int, 2> because ArrayInStruct::field is interpreted as such
+		value_array<std::array<int, 2>>("array_int_2")
+			.element(index<0>())
+			.element(index<1>())
```

Comment:
It might be nice in the future to avoid the `.element(index<...>())` boilerplate by providing shorthand that understands `std::array::size()`